lib/workers/repository/update/pr/changelog/release-notes.ts

```diff
@@ -179,13 +181,17 @@ async function releaseNotesResult(
   }
   const { baseUrl, repository } = project;
   const releaseNotes: ChangeLogNotes = releaseMatch;
-  if (detectPlatform(baseUrl) === 'gitlab') {
-    releaseNotes.url = `${baseUrl}${repository}/tags/${releaseMatch.tag!}`;
-  } else {
-    releaseNotes.url = releaseMatch.url
-      ? releaseMatch.url
-      : /* istanbul ignore next */
-        `${baseUrl}${repository}/releases/${releaseMatch.tag!}`;
+  const platform = detectPlatform(baseUrl);
```

Comment:
this refactor should be done separate

Reply:
Good point - I did this in anticipation of adding a case for `bitbucket`, but looks like I didn't.  Reverting